content/en/developers/marketplace/_index.md

```diff
@@ -3,332 +3,95 @@ title: Marketplace
+* Reading the documentation about setting up an [OAuth 2.0 client][11] for Agent-based or API-based integrations and Datadog Apps.
 
-### Submit a pull request
+## List an offering on Marketplace
 
-Submit a pull request containing all of your files either to the [Marketplace repository][10], or to [Integrations Extras repository][11]. 
+All Technology Partners can list a free integration on the **Integrations** page or a commercial offering on the **Marketplace** page. Additional offerings on Datadog Marketplace may include:
 
-Each repository runs automatic tests to verify that your pull request is in good shape. You can run these same tests locally with the command `ddev validate all`. Once the PR is passing all checks, the Datadog Engineering team reviews to identify blockers to release and makes suggestions around best practices.
+Integrations
+: Integrations that submit or pull third-party data through the Datadog Agent or an API.
 
-If you need access to Azure DevOps for the Marketplace repository, leave a comment in the PR to request access. 
+Software licenses
+: SaaS licenses enable you to deliver and license software solutions to customers through the Datadog Marketplace.
 
-### Approve tile for release
+UI extensions or applications
+: An application (such as a [Datadog App][5]) that are tile-only listings on the Datadog Marketplace.
 
-When the Datadog Engineering and Product teams approve your pull-request, the tile is enabled for your sandbox account. This allows you to validate and view your tile in the Datadog Marketplace and make changes before it goes live.
+Professional services
+: Professional Services enable you to offer your team's services for implementation, support, or management for a set period of time.
 
-## Coordinate go-to-market opportunities
+### Review the example pull request
 
-Once a Marketplace tile is live, Technology Partners have the option of meeting with Datadog's Partner Marketing Team to coordinate a joint go-to-market strategy, including:
+While the process for building an integration is identical for the Integrations page through the [`integrations-extras`][12] repository and the Marketplace page through the [`marketplace`][13] repository, a Marketplace offering requires additional files and information such as pricing plans.
 
-* A Datadog quote for partner press releases
-* A Blog Post in the [Datadog Monitor][34]
-* Amplification of social media posts
+### Request access to Marketplace
 
-## Contact
+To request access to the private Marketplace repository, email <a href="mailto:marketplace@datadoghq.com">marketplace@datadoghq.com</a>.
 
-Email marketplace@datadog.com if you have any questions.
+Once you have been granted access, you can review an [example pull request][14] in the Marketplace repository with annotations and best practices.
 
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
```

Comment:
@alai97 Can we include a link to child page 'Develop a Marketplace offering' in this section?